xml/repeated-content-decl.ent

```diff
@@ -2015,3 +2015,12 @@ User:           username@ldap.example.com
    information see <xref linkend="cha-cap-depl-notes-releases"/>.
   </para>
  </warning>'>
+
+<!--ENTITY values-file-changes...............................................-->
+
+<!ENTITY values-file-changes
+'<para xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+  The format of the &values-file; file has been restructured completely.
+  Do not re-use the previous version of file. Instead, source the default file at
+  <link xlink:href="https://github.com/cloudfoundry-incubator/kubecf/blob/master/deploy/helm/kubecf/values.yaml"/>.
```

Comment:
Going to merge for now, but we should change it to link to our appendix containing the values.yaml file instead. Contents of master will change and we won't need to worry about changes to the URL itself.

Also pending is the decision whether we should point to the values.yaml to sample-values.yaml from the bundle (relevant to what the appendix will contain and not this PR specifically)